### ISSUE TITLE
Refactor to monorepo

### ISSUE THREAD

**Issue by cmaycumber** [nandorojo/redrip-ideas#2]
Is there any particular structure the monorepo should have? 

There are quite a few free-floating files, curious where you think everything should fall.

**Comment by nandorojo** [comment]
Prettier, TS config and linter should all be in the same root.

All the `.tsx` files outside of `src` should go in the example app, these are all just ideas.

**Comment by nandorojo** [comment]
This repo is currently just an `expo init` for easy testing. The `src` folder would be the main package in `packages/redrip` (assuming this name sticks)

**Comment by cmaycumber** [comment]
Sweet I'll create a PR today and we can tweak it as we move along

**Comment by nandorojo** [comment]
Sounds good! I believe I published `@nandorojo/redrip` from a different repo. I was testing in a bunch of different repos, but we can stick to this one.

**Comment by nandorojo** [comment]
@cmaycumber just curious, is this something you're free to take on soon/have planned to? There are a few features/changes I'd like to make, but I can wait if you're turning it into a monorepo now to not get in the way. My app is fully on SDK40 so I'm ready to go with this package.

**Comment by cmaycumber** [comment]
I should be able to get it finished tonight! I haven't started yet though. I can wait if it's something you want to get in immediately

**Comment by nandorojo** [comment]
Tonight sounds good 👌

**Comment by cmaycumber** [comment]
I had this converted over but the last thing I'm running into is a weird error with the bundler on web. I'm getting `Uncaught SyntaxError: Unexpected token ','`. I've managed to track the bug into the `useAnimateToStyle` hook but it's a pretty huge file, so I haven't been able to find it yet.

Any ideas?

**Comment by nandorojo** [comment]
Hm yeah I see the same issue on web. Does this happen in the current repo, or if you just copy the `src` folder over into a new one?

I'm seeing the error on web (works on iOS) in this current repo. Weird that there isn't a better stack trace...

**Comment by cmaycumber** [comment]
I can try to run it in the current repo. 

I'm currently running it out of an example folder with the webpack pointing to the `src/index.tsx` file of the `redrip` package.

**Comment by nandorojo** [comment]
This is happening for me on web even with an empty component in `App.tsx` in the current repo. I wonder if it's something outside of the source code.

**Comment by cmaycumber** [comment]
Yeah, that's pretty odd. I can get mine running by commenting out the `useMapAnimateToStyle` in the redripify function on both iOS and Web; but as soon as I add it back in I get the error.

I thought it might be something to do with `framer-motion` having some special babel config or something but commenting it out hasn't helped me.

**Comment by cmaycumber** [comment]
If the other repo you have is working I can try to port the code over real quick and see if I can get it running.

**Comment by cmaycumber** [comment]
I think I found it.

**Comment by cmaycumber** [comment]
Found it forreal this time it's in this block of code: 

```jsx
if (isColor(key)) {
        if (__DEV__) {
          if (
            typeof value === 'string' &&
            !value.startsWith('rgb') &&
            !value.startsWith('#')
          ) {
            console.error(
              `[${PackageName}]: You passed ${key}: ${value}, but not all color values are supported yet in Reanimated 2. ☹️

      Please use an rgb or hex formatted color.

        Please go to https://github.com/software-mansion/react-native-reanimated/issues/845 and comment so that this bug can get fixed!`
            )
          }
        }
        console.log('[color]', { key, value })
      }
```

**Comment by cmaycumber** [comment]
I think it's because `__DEV__` isn't defined in the browser...

The stack trace on that one was pretty bad.

**Comment by nandorojo** [comment]
Good catch, that's been plaguing me.

**Comment by cmaycumber** [comment]
Monorepo is merged into master in #3
